### PR TITLE
Minor doc cleanup

### PR DESCRIPTION
**Pull request by reyang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,13 @@ The .NET [OpenTelemetry](https://opentelemetry.io/) client.
...
 
-This repository includes multiple installable packages. The `OpenTelemetry.Api`
-package includes abstract classes and no-op implementations for the
-[OpenTelemetry API
-specification](https://github.com/open-telemetry/opentelemetry-specification).
+This repository includes multiple installable packages, available on
+[NuGet](https://www.nuget.org/profiles/OpenTelemetry):
 
-The `OpenTelemetry` package is the reference implementation of the API.
+* The `OpenTelemetry.Api` package includes abstract classes and no-op
```
Comment (cijothomas) [inline README.md:17]: Since the SHIM is also part of this package, should we remove the term "abstract classes"

Comment (reyang) [reply README.md:17]: Let me rephrase this.

Comment (reyang) [reply README.md]: Updated the wording.

File: README.md [hunk 2]
```diff
@@ -9,34 +9,31 @@ Status](https://github.com/open-telemetry/opentelemetry-dotnet/workflows/.NET%20
...
-implements the API.
-
-**Please note** that this library is currently in _alpha_, and shouldn't
-generally be used in production environments.
-
-## Documentation
+Here are the most commonly used components:
 
 * [OpenTelemetry .NET API](./src/OpenTelemetry.Api/README.md)
 * [OpenTelemetry .NET SDK](./docs/sdk-usage.md)
```
Comment (cijothomas) [inline README.md:24]: why dont we move even sdk-usage to the readme.md under SDK project? same with instrumentation?

Comment (reyang) [reply README.md:24]: next PR :)
